llumnix/entrypoints/utils.py

```diff
@@ -27,21 +27,37 @@
 from llumnix.utils import random_uuid
 from llumnix.arg_utils import EngineManagerArgs
 from llumnix.queue.queue_type import QueueType
+from llumnix.server_info import ServerInfo, RequestTimestamps
+from llumnix.queue.utils import init_output_queue_server
 
-logger = init_logger(__name__)
+logger = init_logger("llumnix.api_server")
```

Comment:
why change

Reply:
Does the logger can take effects when 'logger = init_logger(__name__)'?

Reply:
you can give it a try :)